docs/website/docs/deployment-configurations/bare-metal.md

```diff
@@ -0,0 +1,108 @@
+# Run on a Bare-Metal Platform
+
+IREE supports CPU model execution on a bare-metal platform. That is, a platform
+without operating system support and the executable runs on the machine mode.
```

Comment:
can you reword or clarify "and the executable runs on the machine mode"?

Reply:
Reword the paragraph.